editors/code/src/client.ts

```diff
@@ -348,6 +348,7 @@ class ExperimentalFeatures implements lc.StaticFeature {
                     "rust-analyzer.showReferences",
                     "rust-analyzer.gotoLocation",
                     "editor.action.triggerParameterHints",
+                    "editor.action.rename",
```

Comment:
I imagine this would require also adding a comment to #4604
announcing this change, but I'm not 100% sure on the procedure for this.

Reply:
done